.github/workflows/auto_test.yml

```diff
@@ -88,7 +88,7 @@ jobs:
     - name: Building MAM4xx (${{ matrix.build-type }}, ${{ matrix.fp-precision }} precision)
       run: |
         cd build
-        make -j
+        make
```

Comment:
Very odd... Not sure it's worth anyone's time to debug this anyway since we can't reproduce locally, but perhaps some CMake wizardry would fix things. It's also possible constraining the number of parallel jobs within the actions runner to some nominal n<64 or something would fix it, but again, not worth the time testing.